Cargo.toml

```diff
@@ -17,6 +17,12 @@ name = "sccache"
 name = "sccache-dist"
 required-features = ["dist-server"]
 
+[profile.release]
+codegen-units = 1
+lto = true
+strip = true
+panic = "abort"
```

Comment:
Could you explain why we're not keeping backtraces? The performance overhead exists, but compared to cache fetches and launching other binaries, it should be rather negligible (gut feeling)?